src/index.tsx

```diff
@@ -8,12 +8,7 @@ import gtmConfig from './utils/googleTagManager'
 
 TagManager.initialize(gtmConfig)
 
-ReactDOM.render(
-  <React.StrictMode>
```

Comment:
what's the intended effect of removing strict mode?

Reply:
just to remove browser console issue of unsafe_componentunmount warning.
actually it's ok to remain as it or remove.